test/functional/normal/put_spec.lua

```diff
@@ -15,7 +15,7 @@ local dedent = helpers.dedent
 local getreg = funcs.getreg
 
 local function reset()
-  clear()
+  execute('enew!')
```

Comment:
> For clarity, let's put it in a setup() inside the describe() block.

I can have a `setup()` in the `describe()` block, but I still need the `clear()` as a call outside of that to force the connection (so I can call `funcs.split()` for some text pre-processing).
Would you want it added (so it's like the below)?
```lua
clear()
setup(clear)
```